### PR TITLE
More buildscript updates

### PR DESCRIPTION
**Pull request by modmuss50**
Replace stitch with loom and mapping-io

There is more to do after this, but I want to take it one step at a time to make it easier to figure out what I break 😆 .

### CODE REVIEW

**Comment by modmuss50** [comment]
This seems to be causing mapping poet to fail 🤔 

```
Exception in thread "main" java.lang.RuntimeException: Could not find parent class: net/minecraft/class_6740 for net/minecraft/class_6740$class_6741
	at net.fabricmc.mappingpoet.Main.writeClass(Main.java:247)
	at net.fabricmc.mappingpoet.Main.lambda$generate$0(Main.java:95)
	at net.fabricmc.mappingpoet.Main.lambda$forEachClass$5(Main.java:171)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at net.fabricmc.mappingpoet.Main.forEachClass(Main.java:171)
	at net.fabricmc.mappingpoet.Main.generate(Main.java:95)
```

**Review by liach — APPROVED** [review]

**Comment by liach** [comment]
Quick question, does filament have any license?

File: gradle.properties [hunk 1]
```diff
@@ -18,5 +16,6 @@ mappingpoet_version=0.3.2
 
 # Build logic
 tiny_mappings_parser_version=0.3.0+build.17
```
Comment (liach) [inline gradle.properties]: Can be removed as well

Comment (modmuss50) [reply gradle.properties]: Done 👍

**Comment by liach** [comment]
Can you do us a favor and add `mavenLocal()` to filament's repositories as well? That way I just need to update mappingpoet version in gradle.properties only in order to test +local builds.

**Comment by modmuss50** [comment]
> Can you do us a favor and add `mavenLocal()` to filament's repositories as well? That way I just need to update mappingpoet version in gradle.properties only in order to test +local builds.

Done 👍

**Comment by modmuss50** [comment]
Ill try and get this tested and merged quite soon. I had hoped we would have a few more weeks of snapshots where breaking stuff wasnt so bad. 👍

**Comment by modmuss50** [comment]
Tested with Fabric API, everything seems to work OK. Please let me know ASAP if you have any issues after I merge this 👍

**Pull request merged**